From 13fc7d48d45eae07b84fa942083d6e7ef65094d1 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Thu, 7 Mar 2019 15:33:09 +0800
Subject: [PATCH 2/4] sdk_dpaa: assign a value for uninitialized _errno

To avoid below warning:

drivers/net/ethernet/freescale/sdk_dpaa/dpaa_ethtool.c:109:9: warning: '_errno'
may be used uninitialized in this function [-Wmaybe-uninitialized]
  return _errno;
           ^~~~~~

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 .../net/ethernet/freescale/sdk_dpaa/dpaa_ethtool.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_ethtool.c b/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_ethtool.c
index df0a64a..0c632b8 100644
--- a/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_ethtool.c
+++ b/drivers/net/ethernet/freescale/sdk_dpaa/dpaa_ethtool.c
@@ -90,7 +90,7 @@
 static int __cold dpa_get_ksettings(struct net_device *net_dev,
 		struct ethtool_link_ksettings *cmd)
 {
-	int			 _errno;
+	int			 _errno = 0;
 	struct dpa_priv_s	*priv;
 
 	priv = netdev_priv(net_dev);
-- 
1.7.5.4

